### ISSUE TITLE
NVDA fails to report set row and column headers in Excel, even if they are set from NVDA itself

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10541]
### Steps to reproduce:
1. Launch NVDA.
2. Open a workbook in Excel. The workbook has to have at least a few rows and columns filled with some data. In my case the file is an XLSX one and not a XLS (I don't know if it matters).
3. Move the focus to the first row and column with filled data.
4. Press NVDA+Shift+C to set this cell as the first header for all columns from top to bottom and left to right.
5. Use the arrow keys to navigate between cells in the rows and columns of data in the workbook.

### Actual behavior:
NVDA does not report the set column header(s) when switching between columns.

### Expected behavior:
NVDA should report the set column header(s) when switching between columns.

### System configuration
- Operating system: Windows 10 Pro version 1909 (build 18363.476), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19266,67bf1df6, in bulgarian.
- Microsoft Excel version: Microsoft Excel for Office 365, 32-bit, MSO (16.0.12130.20232), click to launch, in bulgarian (information taken from the "About Excel" dialog).
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - 2019.2.1 - the behavior is the same.

**Comment by k-kolev1985** [comment]
I've solved the problem! 😊

In my NVDA configuration, in "Document formatting" section, in the "Table information" grouping, the first check box ("Tables") was unchecked. When I checked it, NVDA correctly reported the set column headers, but at the end of the spoken string (after the cell's content and address). That is OK though. Should I close the issue in this case?

**Comment by leonardder** [comment]
Yes, I think so. Thanks!
